discovery-provider/ddl/migrations/0021_fix_album_repost_save.sql

```diff
@@ -0,0 +1,88 @@
+begin;
+
+-- only use repost type playlist / track
+update saves
+set
+    save_type = 'playlist'
+where
+    save_type = 'album'
+    and (
+        is_current = false
+        or is_current = true
```

Comment:
whats the point of these (is_current=false or us_current=true) conditions?

Reply:
I think the PKEY includes is_current, so the theory is including a WHERE query for it (even if it's for both states, true and false) allows the PKEY to be used when querying using only the secondary part of the composite key